plugin/trino-queryeventlistener/src/main/java/io/trino/plugin/queryeventlistener/model/CompletedEvent.java

```diff
@@ -0,0 +1,218 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package io.trino.plugin.queryeventlistener.model;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+import com.fasterxml.jackson.annotation.JsonProperty;
+import io.trino.spi.eventlistener.QueryCompletedEvent;
+
+public class CompletedEvent
+{
+    private static final String KEY_EVENT_TYPE = "eventType";
+    private static final String KEY_QUERY_ID = "queryId";
+    private static final String KEY_CREATE_TIME = "createTime";
+    private static final String KEY_QUEUED_TIME = "queuedTime";
+    private static final String KEY_WALL_TIME = "wallTime";
+    private static final String KEY_CPU_TIME = "cpuTime";
+    private static final String KEY_USER = "user";
+    private static final String KEY_SCHEMA = "schema";
+    private static final String KEY_CATALOG = "catalog";
+    private static final String KEY_RECORDS = "records";
+    private static final String KEY_COMPLETED = "completed";
+    private static final String KEY_SQL = "sql";
+    private static final String KEY_USER_AGENT = "userAgent";
+
+    @JsonProperty(KEY_EVENT_TYPE)
+    private final String eventType = "QueryCompleted";
+
+    @JsonProperty(KEY_QUERY_ID)
+    private final String queryId;
+
+    @JsonProperty(KEY_CREATE_TIME)
+    private final String createTime;
+
```

Comment:
I feel like maybe we should include the time the query was completed as well? So we don't always have to do the math...